addons/fastroping/data/sounds/LICENSE.md

```diff
@@ -0,0 +1,4 @@
+Uses a sound from Tessa Elieff (Freesound.org)
```

Comment:
This is my format of doing it, adapted from some general guidelines about licensing (https://github.com/Theseus-Aegis/tac-a3-mods/blob/master/addons/radios/sounds/LICENSES.md):
```
**Name (fileName.ogg):**  
"[Title](url)" by [author](authorUrl) is licenseType / changesHere
```